code/game/machinery/tcomms/core.dm

```diff
@@ -13,7 +13,7 @@
 /obj/machinery/tcomms/core
 	name = "Telecommunications Core"
 	desc = "A large rack full of communications equipment. Looks important."
-	icon_state = "core"
+	icon_state = "hub"
```

Comment:
I want to see if I can get the core sprite I did up to TG standards before nuking it entirely